src/rpc/connection/dispatch.c

```diff
@@ -199,13 +198,14 @@ int handle_run(connection_request_event_info *info)
     return (-1);
   }
 
-  if (!meta->obj[0].data.string.str) {
+  if (!meta->obj[0].data.string.str || meta->obj[0].data.string.length != 16) {
```

Comment:
I don't like the idea of not checking the length of the user input. It's not worth processing it if the length does not match. Replaced the `16` with the corresponding macro:  4420ab56ca24ee8a5d9ebe49fb50cc0b5a4e4206